src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -1266,6 +1305,20 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
     }
   }
 
+  private markYourDonationStepIncomplete() {
+    const step = this.stepper.steps.get(0);
+    if (step) {
```

Comment:
Step should always be truth shouldn't it? Can we throw an error if it isn't?

Reply:
It should, yes. Do you think this works? https://github.com/thebiggive/donate-frontend/pull/1415/commits/3b1fce2667541d52a73186ae42d451080292454b